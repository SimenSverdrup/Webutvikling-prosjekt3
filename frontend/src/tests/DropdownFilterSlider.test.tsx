import React from 'react';
import { render } from '@testing-library/react' // ES6
import LongMenu from '../../src/components/Dropdown/dropdownFilterSlider';



it('renders Menu correctly ', () => {
    const { container } = render(<LongMenu />);
    expect(container).toMatchSnapshot();
});
