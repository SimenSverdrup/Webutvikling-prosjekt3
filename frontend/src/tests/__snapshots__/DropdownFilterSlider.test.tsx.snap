// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Menu correctly  1`] = `
<div>
  <div>
    <button
      aria-haspopup="true"
      class="MuiButtonBase-root MuiButton-root MuiButton-text"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        Filter:
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </div>
</div>
`;
